geometry_msgs/Pose center_pose
geometry_msgs/Pose target_pose
geometry_msgs/Pose[] center_poses
geometry_msgs/Pose[] target_poses
